config/api.go

```diff
@@ -52,14 +31,21 @@ type API struct {
 	WebsocketAddress string `json:"websocketaddress"`
 }
 
+// SetArbitrary is an interface implementation of base/fill/struct in order to safely
```

Comment:
This allows us now to parse configs that have more fields than required/defined in the struct. 
On the positive side: it eases the migrations a lot as it's much "softer" on loading and doesn't error out.
On the negative side: It's much easier to miss future errors when you have a typo in some config name as it gets ignored